Sources/KSCrashRecording/Monitors/KSCrashMonitor_User.c

```diff
@@ -43,15 +44,18 @@ static volatile bool g_isEnabled = false;
 
 static KSCrash_ExceptionHandlerCallbacks g_callbacks;
 
-void kscm_reportUserException(const char *name, const char *reason, const char *language, const char *lineOfCode,
-                              const char *stackTrace, bool logAllThreads,
-                              bool terminateProgram) KS_KEEP_FUNCTION_IN_STACKTRACE
+int64_t kscm_reportUserException(const char *name, const char *reason, const char *language, const char *lineOfCode,
+                                 const char *stackTrace, bool logAllThreads,
+                                 bool terminateProgram) KS_KEEP_FUNCTION_IN_STACKTRACE
 {
     if (!g_isEnabled) {
         KSLOG_WARN("User-reported exception monitor is not installed. Exception has not been recorded.");
-        return;
+        return -1;
```

Comment:
`-1` is not very Swift-friendly. What else could we come up with that would bridge nicely to Swift?